src/transformers/models/ibert/configuration_ibert.py

```diff
@@ -123,3 +127,22 @@ def __init__(
         self.position_embedding_type = position_embedding_type
         self.quant_mode = quant_mode
         self.force_dequant = force_dequant
+
+
+class IBertOnnxConfig(OnnxConfigWithPast):
```

Comment:
For encoder-based models like IBERT, I think it makes more sense to use `OnnxConfig` instead of `OnnxConfigWithPast`. The `OnnxConfigWithPast` is usually reserved for decoder-based and encoder-decoder models where having access to the past key values is important for text generation.

Is there a reason why you chose this particular config?

Reply:
I followed the recommendations in this [tutorial](https://huggingface.co/docs/transformers/serialization#converting-an-onnx-model-using-the-transformersonnx-package). In particular, I was looking at this [patch](https://github.com/huggingface/transformers/pull/13049/commits/d097adcebd89a520f04352eb215a85916934204f).

Reply:
Thanks for the information @MaximovaIrina - we're in the process of revamping the docs / tutorial, so hopefully it will be less confusing in future!

The patch you linked to is for MBART, which is an encoder-decoder model where having access to past key values is important. In your case with IBERT, I recommend looking at the [DistilBERT example](https://github.com/huggingface/transformers/blob/6b655cc63fd7e8ec120d9c38321a286d04767db0/src/transformers/models/distilbert/configuration_distilbert.py#L136) which inherits from `OnnxConfig`.

As @michaelbenayoun notes elsewhere, you can also use this as a guide for adding all the supported features.